packages/documentation/.storybook/components/docs/layout.js

```diff
@@ -0,0 +1,91 @@
+import pkgDocumentation from './../../../package.json';
+import pkgStyles from './../../../node_modules/@swisspost/design-system-styles/package.json';
+import React from "react";
+
+import { DocsContainer as Layout } from "@storybook/addon-docs/blocks";
+import Header from './header';
+import TopicTeaser from './topic-teaser';
+import Footer from './footer';
+import './layout.scss';
+
+import { useDarkMode } from "storybook-dark-mode";
+import postThemes from '../../post-themes';
+
+const versionFilterRegexes = {
+  major: /^(?:(\d+)\.\d+\.\d+)/,
```

Comment:
Here you can use [named capturing groups](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Groups_and_Backreferences#using_named_groups) so that the RegExp is only defined once. 

It would also simplify the `getVersion` function since then you could use the [groups object ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec#groups) returned in the result of the regex `.exec()` method

Reply:
As every regex is a little different, sonar is happy with it and we allready use these exact regexes in the demo app. I would prefer to not refacter the hole logic of `getVersion`, because of this.
If you can not live with that, please provide a code suggestion how you would write the given code.

Reply:
It was a good opportunity to update the code with new JS features, but the current logic is correct so I can "live with that" for sure!